packages/fxa-settings/src/components/FlowContainer/index.tsx

```diff
@@ -9,12 +9,16 @@ import { ReactComponent as BackArrow } from './back-arrow.svg';
 type FlowContainerProps = {
   title?: string;
   subtitle?: string;
+  onBackButtonClick?: (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => void;
```

Comment:
This is in https://github.com/mozilla/fxa/pull/6938.  Please review that first if that's not yet merged.